src/pretix/static/pretixpresale/scss/_cart.scss

```diff
@@ -94,23 +95,88 @@
     }
 }
 
-.apply-voucher {
-    input {
-        height: 32px;
+.cart [role=rowgroup]:last-child {
+    border-top: 1px solid $table-border-color;
+}
+.cart .total strong {
+    font-size: $font-size-large;
```

Comment:
I generally like the larger total, but the font sizes and paddings become a bit messy if you turn "display net prices":

![20210929_170641](https://user-images.githubusercontent.com/64280/135299387-6ec83233-130b-4d2e-8b9b-644cc53c1e8e.png)

Reply:
Yes, probably the text „Net total“ should not be that big. Or the „€ 110“ should be big as well. Will try both.

Reply:
I think making „Net total“ as big/small as the other text and only „Total“ bigger is best.

![Bildschirmfoto 2021-09-29 um 22 03 46](https://user-images.githubusercontent.com/276509/135340538-f21cc3c9-c6f2-4c65-b5b9-19cd25452934.png)

Reply:
agreed!